fuzzers/baby_fuzzer/baby_fuzzer.py

```diff
@@ -0,0 +1,70 @@
+from pylibafl import libafl
+
+# LIBRARY WRAPPER
+
+def map_observer_wrapper(map_observer):
+    if type(map_observer).__name__ == "OwnedMapObserverI32":
+        return libafl.MapObserverI32.new_from_owned(map_observer)
+
+def executor_wrapper(executor):
+    if type(executor).__name__ == "OwnedInProcessExecutorI32":
+        return libafl.ExecutorI32.new_from_inprocess(executor)
+
+def monitor_wrapper(monitor):
+    if type(monitor).__name__ == "SimpleMonitor":
+        return libafl.Monitor.new_from_simple(monitor)
+
+def event_manager_wrapper(event_manager):
+    if type(event_manager).__name__ == "SimpleEventManager":
+        return libafl.EventManagerI32.new_from_simple(event_manager)
+
+def corpus_wrapper(corpus):
+    if type(corpus).__name__ == "InMemoryCorpus":
+        return libafl.Corpus.new_from_in_memory(corpus)
+    if type(corpus).__name__ == "OnDiskCorpus":
+        return libafl.Corpus.new_from_on_disk(corpus)
+
+def rand_wrapper(rand):
+    if type(rand).__name__ == "StdRand":
+        return libafl.Rand.new_from_std(rand)
+
+def stage_wrapper(stage):
+    if type(stage).__name__ == "StdScheduledHavocMutationsStageI32":
+        return libafl.StageI32.new_from_std_scheduled(stage)
```

Comment:
Why `I32`?

Reply:
These wrappers should be shipped with the module (not written by the user). 
This is just a PoC. Ideally, we would have all the types (I8,I16, ..., U64)

Reply:
> Why `I32`?

No generics support on pyo3